src/test/java/org/fisco/bcos/channel/test/parallel/precompile/transferSign.java

```diff
@@ -0,0 +1,61 @@
+package org.fisco.bcos.channel.test.parallel.precompile;
+
+import java.math.BigInteger;
+import org.fisco.bcos.web3j.crypto.EncryptType;
+import org.fisco.bcos.web3j.crypto.gm.sm2.SM2Sign;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class transferSign {
+    private static Logger logger = LoggerFactory.getLogger(transferSign.class);
+
+    public static void Usage() {
+        System.out.println(" Usage:");
+        System.out.println(
+                " \t java -cp conf/:lib/*:apps/* org.fisco.bcos.channel.test.parallel.parallelok.transferSign txCount ThreadCount GMOrNot CacheOrNot.");
+        System.exit(0);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [System.exit() should not be used in J2EE/JEE apps](https://app.codacy.com/manual/fisco/web3sdk/pullRequest?prid=4981260)